### ISSUE TITLE
Detector data cleanup

### ISSUE THREAD

**Issue by JeremyMcCormick** [JeffersonLab/hps-java#142]
- Generate LCDD files for detectors where they are missing

- Make sure all detectors have sampling fraction data and correct detector.properties file

- Regenerate LCDD files using current versions of the converters

**Comment by JeremyMcCormick** [comment]
```
Exception in thread "main" java.lang.RuntimeException: java.lang.RuntimeException: Layer 4 is not defined.
        at org.lcsim.geometry.compact.converter.Main.main(Main.java:107)
Caused by: java.lang.RuntimeException: Layer 4 is not defined.
        at org.lcsim.geometry.compact.converter.HPSTestRunTracker2014GeometryDefinition$TestRunHalfModuleStereo.<init>(HPSTestRunTracker2014GeometryDefinition.java:1487)
        at org.lcsim.geometry.compact.converter.HPSTestRunTracker2014GeometryDefinition.createTestRunHalfModuleStereo(HPSTestRunTracker2014GeometryDefinition.java:2355)
        at org.lcsim.geometry.compact.converter.HPSTestRunTracker2014GeometryDefinition.createTestRunHalfModuleStereo(HPSTestRunTracker2014GeometryDefinition.java:22)
        at org.lcsim.geometry.compact.converter.HPSTrackerGeometryDefinition.makeHalfModule(HPSTrackerGeometryDefinition.java:127)
        at org.lcsim.geometry.compact.converter.HPSTestRunTracker2014GeometryDefinition.makeModuleBundle(HPSTestRunTracker2014GeometryDefinition.java:179)
        at org.lcsim.geometry.compact.converter.HPSTestRunTracker2014GeometryDefinition.build(HPSTestRunTracker2014GeometryDefinition.java:110)
        at org.lcsim.geometry.compact.converter.HPSTestRunTracker2014LCDDBuilder.build(HPSTestRunTracker2014LCDDBuilder.java:47)
        at org.lcsim.geometry.compact.converter.lcdd.HPSTracker2014Base.addToLCDD(HPSTracker2014Base.java:105)
        at org.lcsim.geometry.compact.converter.lcdd.LCDDDetector.writeSubdetectors(LCDDDetector.java:214)
        at org.lcsim.geometry.compact.converter.lcdd.LCDDDetector.writeLCDD(LCDDDetector.java:133)
        at org.lcsim.geometry.compact.converter.lcdd.Main.convert(Main.java:63)
        at org.lcsim.geometry.compact.converter.Main.run(Main.java:115)
        at org.lcsim.geometry.compact.converter.Main.main(Main.java:103)
```

**Comment by JeremyMcCormick** [comment]
Resolved by PR #192.
